dynamicweb/settings/base.py

```diff
@@ -526,9 +521,31 @@ def bool_env(val):
     'dynamicweb-staging.ungleich.ch': 'staging'
 }
 
+ENABLE_LOGGING = bool_env('ENABLE_LOGGING')
```

Comment:
I suggest to call the environment variable ENABLE_DEBUG_LOGGING, as this is more precise what it is doing